sphinx/texinputs/Makefile

```diff
@@ -63,8 +71,8 @@ bz2: tar
 	$(PDFLATEX) $(LATEXOPTS) '$<'
 
 clean:
-	rm -f *.dvi *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla
+	rm -f *.dvi *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.pdf *.tar *.tar.gz *.tar.bz2 *.tar.xz
```

Comment:
It should not remove all of .pdf files. because it might be included the images from users.
It was once implemented, but removed at 2e5bb8136a0715396c2c4d34d7c76a69742fe0a4

I think `$(ALLPDF)` is better.